Add Identity and Access Management into HCC settings procedure. (#642)
redhat-developer/app-services-guides: docs/kafka/access-mgmt-kafka/README.adoc

.Procedure

> . In the toolbar of the {service-url-kafka}[Openshift Streams for {product-kafka} web console], select the gear icon.

Reviewer: ```suggestion
. In the toolbar of the {service-url-kafka}[{product-kafka} web console], select the gear icon.
```

.Procedure

> . In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*

Reviewer: ```suggestion
. In the toolbar of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*.
```

Author: @smccarthy-ie I do not think we need to click "Settings" anymore?
Also, I think both guides should read the same first line and maybe Kafka should also say 
"At the top of the [Streams for Apache Kafka web console](https://console.redhat.com/application-services/streams/kafkas), select the gear icon."

Reviewer: @fmcdonal Good spot! Yes should be "click *Identity & Access Management* > *User Access* > *Groups*" :)

. Click the name of the user group.
. From the *Roles* tab, click *Add role* and select `User Access principal viewer`.

> . Click *Add to group* to add the role to the group.

Reviewer: ```suggestion
. Click *Add to group*.
```

Author: @smccarthy-ie separating this into 2 steps like in registry.
